xml/System.IO.Enumeration/FileSystemEnumerable`1.xml

```diff
@@ -35,7 +35,17 @@
   <Docs>
     <typeparam name="TResult">The type that this enumerable encapsulates.</typeparam>
     <summary>Allows utilizing custom filter predicates and transform delegates for enumeration purposes.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
```

Comment:
I decided to put the example in the remarks of the type, because I think it's the most visible place.